resources/views/login.blade.php

```diff
@@ -9,7 +9,7 @@
                 <div class="row justify-content-center">
                     <div class="col-sm-9 col-md-6 col-lg-5 col-xl-4">
 
-                        <h1 class="text-center mb-3">Sharp</h1>
+                        <h1 class="text-center mb-3">{{config('sharp.name')}}</h1>
```

Comment:
Could you use "Sharp" as default here? I mean:
`config("sharp.name", "Sharp")`
to avoid an empty string if the config is not defined.

Reply:
Sure, thank you